include/nlohmann/detail/json_custom_base_class.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <type_traits> // conditional, is_same
+
+namespace nlohmann
+{
+namespace detail
+{
+
+/*!
+@brief Default base class of the @ref basic_json class.
+
+So that the correct implementation of the copy / move ctors / assign operators
+of @ref basic_json does not require complex case distinctions
+(no base class / custom base class), @ref basic_json always a base class.
+By default this class is used, since empty and thus has no effect on the
+behaviour of @ref basic_json.
+
+
+So that the correct implementation of the copy / move ctors / assign operators
```

Comment:
This paragraph seems to be a duplicate of the previous one.

Reply:
Ups. Fixed in 3311d11